addons/stock/views/product_views.xml

```diff
@@ -23,10 +23,13 @@
             </field>
         </record>
 
-        <record model="ir.actions.act_window" id="product_open_orderpoint">
-            <field name="context">{'default_product_id': active_id, 'search_default_product_id': active_id}</field>
-            <field name="name">Reordering Rules</field>
-            <field name="res_model">stock.warehouse.orderpoint</field>
+        <record model="ir.actions.server" id="action_open_routes">
+            <field name="name">Routes</field>
+            <field name="model_id" ref="product.model_product_template"/>
+            <field name="state">code</field>
+            <field name="code">
+                action = model.action_open_routes_diagram()
+            </field>
```

Comment:
not sur it is useful to make a server action for this one

Reply:
For the spec: 'skip wizard if there is only one warehouse'